tests/regression_test.py

```diff
@@ -43,7 +43,7 @@ def make_trajectory(x, y, vx, vy, flux):
     return t
 
 
-def make_fake_image_stack(times, trjs):
+def make_fake_image_stack(times, trjs, psf_vals):
```

Comment:
maybe I'm missing that one of the tests already covers this, but should we have one that tests run_search when a psf_file isn't provided?

Reply:
Good call. I updated the test so roughly 1/3 of the psfs will use the default and 2/3s will use the value from the file. Since the default is used when a file is not provided, I believe this should now cover that case. But let me know if you think it should be extended more.